internal/cli/data/universal-login/assets/index-89540052.js

```diff
@@ -9363,7 +9363,7 @@ function encode_char(c) {
+`+c}try{typeof Object.defineProperty=="function"?Object.defineProperty(e.escapeXML,"toString",{value:p}):e.escapeXML.toString=p}catch{console.warn("Failed to set escapeXML.toString (is the Function prototype frozen?)")}e.shallowCopy=function(g,h){if(h=h||{},g!=null)for(var b in h)a(h,b)&&(b==="__proto__"||b==="constructor"||(g[b]=h[b]));return g},e.shallowCopyFromList=function(g,h,b){if(b=b||[],h=h||{},g!=null)for(var y=0;y<b.length;y++){var _=b[y];if(typeof h[_]<"u"){if(!a(h,_)||_==="__proto__"||_==="constructor")continue;g[_]=h[_]}}return g},e.cache={_data:{},set:function(g,h){this._data[g]=h},get:function(g){return this._data[g]},remove:function(g){delete this._data[g]},reset:function(){this._data={}}},e.hyphenToCamel=function(g){return g.replace(/-[a-z]/g,function(h){return h[1].toUpperCase()})},e.createNullProtoObjWherePossible=function(){return typeof Object.create=="function"?function(){return Object.create(null)}:{__proto__:null}instanceof Object?function(){return{}}:function(){return{__proto__:null}}}()})(j4);const zct="ejs",Pct="Embedded JavaScript templates",Sct=["template","engine","ejs"],Ect="3.1.9",Oct="Matthew Eernisse <mde@fleegix.org> (http://fleegix.org)",Cct="Apache-2.0",Bct={ejs:"./bin/cli.js"},Mct="./lib/ejs.js",Lct="ejs.min.js",Ict="ejs.min.js",Rct={type:"git",url:"git://github.com/mde/ejs.git"},Fct="https://github.com/mde/ejs/issues",Dct="https://github.com/mde/ejs",Vct={jake:"^10.8.5"},Uct={browserify:"^16.5.1",eslint:"^6.8.0","git-directory-deploy":"^1.5.1",jsdoc:"^4.0.2","lru-cache":"^4.0.1",mocha:"^10.2.0","uglify-js":"^3.3.16"},Kct={node:">=0.10.0"},Gct={test:"mocha -u tdd"},qct={name:zct,description:Pct,keywords:Sct,version:Ect,author:Oct,license:Cct,bin:Bct,main:Mct,jsdelivr:Lct,unpkg:Ict,repository:Rct,bugs:Fct,homepage:Dct,dependencies:Vct,devDependencies:Uct,engines:Kct,scripts:Gct};(function(e){/**
```

Comment:
What is this for?

Reply:
Looks like an artifact of the ejs package (https://github.com/mde/ejs/blob/29b076cdbbf3eb1b4323b33299ab6d79391b2c33/lib/utils.js#L119) . Perhaps an artifact of debugging some universal login preview? But yeah, this should be reverted.